scipy/optimize/_qap.py

```diff
@@ -370,19 +381,19 @@ def f(x):  # computing the original optimization function
         np.transpose(cost_matrix) @ dist_matrix[np.ix_(perm_inds, perm_inds)]
     )
 
-    res = {"col_ind": perm_inds, "score": score, "nit": n_iter}
+    res = {"col_ind": perm_inds, "score": score, "nit": total_iter}
```

Comment:
Previously, `nit` was the number of FW iterations for the last initial position. I assumed this should be the total number of iterations, so I added the `total_iter` variable to the code.

Reply:
I think maybe it would make sense for it to return the number of iterations during the initialization resulting in the permutation returned?

Reply:
What can you do with that number?

I've used iteration counts to investigate computational complexity and speed up my code where it makes a difference. With a total iteration count, you can calculate the average over all the initial search positions, which could conceivably be useful.

I don't think either number is terribly useful, but I can't really think of anything that can be done with the number of iterations for the best initial search position. After all, the user doesn't even know what search position that was! What do you have in mind?

Reply:
The `nit` variable is often helpful for us when calibrating the stopping criterion. For example, when running QAP on with different sized graphs if the nit is around 10, then we know that the algorithm likely hasn't reached convergence, and we would lower  the `eps` value.

My thoughts would be to either return the number of iterations during the initialization resulting in the permutation returned or the average iterations among all initializations.

Reply:
OK, then I'd be happy to compromise on the average number of iterations multiplied by the number of initializations : )

Go ahead and leave it as is for now. The most important thing is that the documentation matches. I'll ask for others' opinions.